examples/sdk/sdk-go/go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/Apicurio/apicurio-registry/examples/sdk/sdk-go
+
+go 1.23.0
+
+toolchain go1.24.1
+
+//toolchain go1.24.1
+
+require (
+	github.com/apicurio/apicurio-registry/go-sdk v0.0.0-20241211181742-779f0994a1de // v3.0.6
```

Comment:
TODO: We have to change the go-sdk module name, see https://go.dev/doc/modules/version-numbers#major-version .